packages/flame/lib/src/experimental/geometry/shapes/rounded_rectangle.dart

```diff
@@ -197,7 +197,9 @@ class RoundedRectangle extends Shape {
   @override
   String toString() =>
       'RoundedRectangle([$_left, $_top], [$_right, $_bottom], $_radius)';
-}
 
-@internal
-const tau = Transform2D.tau; // 2π
+  @override
+  Vector2 randomPoint({Random? random, bool within = true}) {
+    throw UnimplementedError();
```

Comment:
you could use the rectangle algo and do the contains check on a while loop, like the arbitrary poly

Reply:
Done! It won't work for `within = false` though.